i18n/en-US.yml

```diff
@@ -337,10 +337,16 @@ components:
       header: Mobility Profile
       mobilityDevices: "Mobility devices: "
       mobilityLimitations: "Mobility limitations: "
+      planTripDescription: >-
+        If you have a travel companion, you can choose to plan this trip
+        according  to their mobility profile. To add travel companions, visit
```

Comment:
At this time, there is no mechanism to add a "dependent" person to an account as this text might suggest (the "dependent" person, for which we are planning trips for, has to add the companion first), so the second part of the label could be removed for now.